cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -79,26 +82,33 @@ case class CommandLineTool private(
         OutputDefinition(FullyQualifiedName(cop_id).id, womType, CommandOutputExpression(outputBinding, womType, inputNames))
     }.toList
 
-    val inputs: List[_ <: Callable.InputDefinition] =
+    val inputDefinitions: List[_ <: Callable.InputDefinition] =
       this.inputs.map { cip =>
-        val tpe = cip.`type`.flatMap(_.select[CwlType]).map(cwlTypeToWdlType).get
-
-        RequiredInputDefinition(FullyQualifiedName(cip.id).id, tpe)
+        val inputType = cip.`type`.flatMap(_.select[CwlType]).map(cwlTypeToWdlType).get
```

Comment:
forcompable?

Reply:
Actually, I think I can tidy it somewhat by just bringing forwards the `.get`.

Of course it'll still be a `.get` but that's a known `TODO`